distributed/scheduler.py

```diff
@@ -2086,6 +2091,16 @@ def _transition(
             return recommendations, client_msgs, worker_msgs
         except Exception:
             logger.exception("Error transitioning %r from %r to %r", key, start, finish)
+            self.log_event(
+                "transistions",
+                {
+                    "action": "transition-failure",
+                    "key": key,
+                    "start": start,
+                    "finish": finish,
+                    "transistion_log": self.transition_log.copy(),
```

Comment:
The transition log can grow quite large so I'm wondering if it makes sense to include it here. Would it be more meaningful to include `Scheduler.story(key, self.transition_log)`?

Reply:
from a chat with @fjetter we estimate logs likely to be worst case a couple of MB so will keep as self.transition_logs.copy() for now